Projeto-Stream/src/views/Home/homeDetail.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+    <div :style="{ backgroundImage: `linear-gradient(to right, black, rgba(0, 0, 0, 0.4)), url(${backgroundUrl})` }"
+        class="bg-fixed bg-center bg-cover h-screen py-24 px-10 grid items-center ">
+        <div>
+            <div class="p-5 w-12/12 md:w-4/12">
+                <h1 class="text-3xl font-base font-bold whitespace-normal text-white">{{ nameOrTitle }}
+                </h1>
+                <div class="flex gap-4 items-center my-2 text-white text-sm">
+                    <p class="font-base font-medium ">{{ firstAirDate }}</p>
+                    <p class="text-green-700 font-base font-bold">{{ averageVote }}</p>
+                    <p class=" font-base font-medium">|</p>
+                    <p class="  font-base font-medium">{{ movieOrTvShow?.runtime }} min</p>
```

Comment:
Try using the <span> tag instead of the <p> tag for short texts. p is more used for long texts